docs/docker/README.MD

```diff
@@ -4,7 +4,17 @@
 Docker is a tool used by developers to package together dependencies into a single container (or image). What this means for *you* is that in order to use your integration, you are not required to "pip install" all of the packages required. They are part of a container that "docks" to the server and contains all of the libraries you need. To learn more about docker, [visit their site here](https://docs.docker.com/)
 
 # Why Use Docker?
-Primarily we use docker to run python scripts in a controlled environment. They run isolated from the server to prevent someone from accidentally damaging the server. By packaging libraries and dependencies together, we can prevent unknown issues from occurring since the environment is all the same.
+Primarily we use docker to run python scripts and integrations in a controlled environment. They run isolated from the server to prevent someone from accidentally damaging the server. By packaging libraries and dependencies together, we can prevent unknown issues from occurring since the environment is all the same.
+
+## Script/Integration Configuration
+Specifying which docker image to use is done in the Demisto IDE (Open: Settings -> Docker image name). If you don't specify a docker image, a default docker image using Python 2.7 is used. For new scripts and integrations, unless there is a specific reason not to use Python 2 (for example: a need to use a library which is not available for Python 3), we require using a Python 3 image. 
```

Comment:
worth mentioning here that it can also be done in the pycharm plugin

Reply:
I think it's ok. I want to keep things short.